src/test/java/org/candlepin/subscriptions/tally/billing/BillableUsageControllerTest.java

```diff
@@ -174,13 +174,13 @@ void monthlyWindowWithRemittanceUpdate() {
   void monthlyWindowRemittanceMultipleOfBillingFactor() {
```

Comment:
The `BillableUsageControllerTest` still contains references to `uom` in:
- method `billable`
- method `createSubscriptionDefinition`